runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -1432,6 +1442,14 @@ private boolean processHerbiboarHerbSackLoot(int timestamp)
 		return true;
 	}
 
+	private void processZombiePirateLockerLoot(Matcher matcher)
+	{
+		List<ItemStack> drops = new ArrayList<>();
+		final int quantity = Integer.parseInt(matcher.group(1) + matcher.group(2));
+		drops.add(new ItemStack(itemManager.search(matcher.group(3)).get(0).getId(), quantity, client.getLocalPlayer().getLocalLocation()));
```

Comment:
`List.of` can be used here as well as  named groups on the regex to be more explicit.

```suggestion
		List.of(new ItemStack(itemManager.search(matcher.group("item")).get(0).getId(), quantity, client.getLocalPlayer().getLocalLocation()));
```